### PR TITLE
logger.package documentation update

### PR DESCRIPTION
**Pull request by burner**
* Headings make stuff easier to find
* Thread local logger description

tab to whitespace

### CODE REVIEW

File: std/experimental/logger/package.d [hunk 1]
```diff
@@ -1,19 +1,29 @@
 /**
 Implements logging facilities.
 
+Copyright: Copyright Robert "burner" Schadek 2013 --
```
Comment (quickfur) [inline std/experimental/logger/package.d]: Why was the copyright notice moved to the top? Usually in Phobos modules they appear toward the bottom of the module docs (before the module members).

**Comment by andralex** [comment]
Auto-merge toggled on

File: std/experimental/logger/package.d (hunk 1 above)
Comment (burner) [inline std/experimental/logger/package.d]: It look quite bad right in the middle

**Comment by burner** [comment]
thanks for reviewing

**Pull request merged**